__tests__/app.test.js

```diff
@@ -67,6 +67,130 @@ describe("/api/reviews", () => {
                 });
             });
     });
+    test("GET - 200: Responds with an array of review objects sorted by date in descending order if no query params are added", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then((res) => {
+                const reviews = res.body.reviews;
+                expect(reviews.length).toBeGreaterThan(0);
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        category: expect.any(String),
```

Comment:
as this is an extension to the above test you dont need to check if each review has all of these keys again as that is being done in the test above already. The same can be done for the test below.